src/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Tools.Common;
+using Microsoft.DotNet.Tools.NuGet;
+
+namespace Microsoft.DotNet.Tools.List.PackageReferences
+{
+    internal class ListPackageReferencesCommand : CommandBase
+    {
+        //The file or directory passed down by the command
+        private readonly string _fileOrDirectory;
+        private AppliedOption _appliedCommand;
+
+        public ListPackageReferencesCommand(
+            AppliedOption appliedCommand,
+            ParseResult parseResult) : base(parseResult)
+        {
+            if (appliedCommand == null)
+            {
+                throw new ArgumentNullException(nameof(appliedCommand));
+            }
+
+            _fileOrDirectory = PathUtility.GetAbsolutePath(PathUtility.EnsureTrailingSlash(Directory.GetCurrentDirectory()),
+                                                           appliedCommand.Arguments.Single());
+
+            _appliedCommand = appliedCommand["package"];
+        }
+
+        public override int Execute()
+        {
+            return NuGetCommand.Run(TransformArgs());
+        }
+
+        private string[] TransformArgs()
+        {
+            var args = new List<string>
+            {
+                "package",
+                "list",
+            };
+
+            args.Add(GetProjectOrSolution());
+
+            args.AddRange(_appliedCommand.OptionValuesToBeForwarded());
+
+            if (_appliedCommand.HasOption("include-prerelease"))
```

Comment:
I think `HasOption` supports the `--` prefix, so perhaps the check for the given option being passed could be combined into `CheckForOutdated`?  Thus just `CheckForOutdated("--include-prerelease")`, etc.  We'll want to check that's the case, though.